ui/de-lib/src/main/java/org/iplantc/de/client/models/tool/ToolVolumesFrom.java

```diff
@@ -0,0 +1,39 @@
+package org.iplantc.de.client.models.tool;
+
+import org.iplantc.de.client.models.HasId;
+
+import com.google.gwt.user.client.ui.HasName;
+import com.google.web.bindery.autobean.shared.AutoBean.PropertyName;
+
+/**
+ * Created by aramsey on 10/30/15.
+ */
+public interface ToolVolumesFrom extends HasName, HasId {
+
+    @PropertyName("id")
+    void setId(String id);
+
+    @PropertyName("tag")
+    void setTag(String tag);
+
+    @PropertyName("tag")
+    String getTag();
+
+    @PropertyName("url")
+    void setUrl(String url);
+
+    @PropertyName("url")
+    String getUrl();
```

Comment:
FYI: The `@PropertyName` annotation isn't required when the method name after `get/set` matches the property name. So `@PropertyName("url")` isn't required for `getUrl`, but `@PropertyName("name_prefix")` is required for `getNamePrefix`